fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/Constants.java

```diff
@@ -19,4 +19,8 @@ public class Constants {
 
   /** Using the Resources class constant would require a min API of 29 */
   public static final int ID_NULL = 0;
+
+  public static final float DISABLED_ALPHA = 0.4f;
```

Comment:
Quick question, why alpha?

Reply:
It's the terminology for transparentness